Packs/CloudIncidentResponse/Triggers/Trigger_-_Cloud_Data_Exfiltration_Response.json

```diff
@@ -0,0 +1,18 @@
+{
+  "trigger_id": "8e153b5ab6f8d93578fa6e19a908efdf",
```

Comment:
Are you sure this is the md5 of 'Cloud Data Exfiltration Response'?

Reply:
forgot I changed the playbook name :) will update to the new MD5